doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,140 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external memory
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide explains how you can use the nRF70 Series firmware patches in the external memory feature.
+
+Overview
+********
+
+By default the nRF70 Series firmware patches are built as part of the nRF Wi-Fi host driver code, residing in on-chip memory.
+The firmware patches include code that is executed on the nRF70 Series device.
+The size of the firmware patches may be considerably large, which limits the amount of on-chip code memory available for the user application.
+In order to increase the amount of on-chip memory available for user applications, the nRF70 Series Wi-Fi driver supports the option of using external memory, if that is available.
+
+Prerequisites
+=============
+
+Before using this feature, make sure that the following prerequisites are completed:
+
+ * The external memory must be available and configured in the devicetree.
+ * The external memory must be large enough to store the firmware patches, taking into consideration also the patch upgrade requirement, for example, Device Firmware Update (DFU).
+   The maximum size of all the firmware patches combined is 128 KB.
```

Comment:
Do we need to keep prefixing `nRF70 Series` everywhere?  Entire page is under `nRF70`.

Reply:
This is very early in the Guide, and it is good to clarify that we are talking about the firmware that runs  on the nRf70 Series device (so not to be confused with the firmware on the host platform). No, we do not need to prefix everything with nRF70 Series - only where there is a risk of confusion.

nRF70 Series is a tricky case. That is because while it is a proper, formal, Nordic Series of devices, it does not work "standalone" :)

Reply:
* Develeoping with nRF70 Series
   - Firmware patches in external memory
      - firmware patch

Based on above heirarchy, IMHO it should be evident :), but I can add if you feel repeating helps.